module/item/item-sheet.js

```diff
@@ -30,7 +30,7 @@ export class PTUItemSheet extends ItemSheet {
 
 		// Alternatively, you could use the following return statement to do a
 		// unique item sheet by type, like `weapon-sheet.html`.
-		return `${path}/item-${this.item.type}-sheet.html`;
+		return `${path}/item-${this.item.type}-sheet.hbs`;
```

Comment:
Can't add it as a suggestion because there were no changes at that point in this file.... but:

At line 159 of this file, add a parameter ownerId, and pass it along during creation

```
	/**
	 * Handle To Chat call.
	 * @private
	 */
	_toChat(ownerId) {
		switch(this.object.type) {
			case "move":
				return sendMoveMessage({
					speaker: ChatMessage.getSpeaker({
						actor: this.actor
					}),
					name: this.object.name,
					move: this.object.system,
					templateType: 'details',
					owner: ownerId
				});
			default: 
				return sendItemMessage({
					speaker: ChatMessage.getSpeaker({
						actor: this.actor
					}),
					item: this.object
					owner: ownerId
				});
		}
	}
```